sql/moz-fx-data-shared-prod/monitoring/suggest_click_rate_live/view.sql

```diff
@@ -5,3 +5,12 @@ SELECT
   *
 FROM
   `moz-fx-data-shared-prod.monitoring_derived.suggest_click_rate_live_v1`
+WHERE
+  DATE(submission_minute) >= DATE_SUB(CURRENT_DATE, INTERVAL 1 DAY)
+UNION ALL
+SELECT
+  *
+FROM
+  `moz-fx-data-shared-prod.monitoring_derived.suggest_click_rate_v1`
```

Comment:
We don't need to add a stable derived table for click rates, since that table has 30 day retention. It's suggest impressions only that potentially have search queries and thus very limited retention.

Reply:
Ah, I see. Thanks Jeff!